apps/portal/lib/graphql/resolvers/OldEvent.resolver.ts

```diff
@@ -0,0 +1,49 @@
+import { Event } from '@generated/type-graphql';
+import { injectable } from 'tsyringe';
+import { Arg, Ctx, FieldResolver, Mutation, Resolver, UseMiddleware } from 'type-graphql';
+import type { TContext } from '../interfaces/context.interface';
+import { InjectSessionMiddleware } from '../middlewares/inject-session';
+import Firebase from '../services/Firebase.service';
+import AdditionalCRUDEventService from '../services/AdditionalCRUDEvent.service';
+import EventCheckinService from '../services/EventCheckin.service';
+import { EventCheckin, EventCheckinInput } from '../schemas/EventCheckin';
+
+@Resolver(() => Event)
+@injectable()
+export default class OldEventResolver {
+  constructor(
+    private eventService: AdditionalCRUDEventService,
+    private checkService: EventCheckinService,
+    private firebase: Firebase,
+  ) {}
+
+  @Mutation(() => [EventCheckin])
+  async checkInOldEvent(
+    @Arg('netID', () => String) netId: string,
+    @Arg('email', () => String) email: string,
+    @Ctx() context: TContext,
+  ) {
```

Comment:
Add return type here.

Reply:
Uhh what exactly do you mean? Don't we have a return type as @Mutation(() =>[EventCheckin])

Reply:
Yes, but the `@Mutation(() => [EventCheckin])` is type annotation for generating GraphQL schema. We also need type annotation for TypeScript function too. If you refer to other resolver functions, you will notice that each function will look something like the following: 
```js
@Mutation(() => ReturnType)
async foo(@Arg("someArg", () => SomeType) someArg: SomeType) : Promise<ReturnType> {
    // do something here
} 
```
In short, we just need the `Promise<ReturnType>` equivalent for the function in question here, which should be `Promise<EventCheckin[]>` in this case.